pkg/server/endpoints/node/handler_test.go

```diff
@@ -153,6 +156,13 @@ func (s *HandlerSuite) SetupTest() {
 		}),
 	})
 	handler.limiter = s.limiter
+	cache, err := lru.New(100)
+	s.Require().NoError(err)
+	s.fetchRegistrationEntriesCache = &regentryutil.FetchRegistrationEntriesCache{
+		Cache:   cache,
+		TimeNow: time.Now,
```

Comment:
Seems like we would want to use a mock clock for testing, i.e. `handler.Clock.Now`

Reply:
updated